scripts/dbSNP/format_dbSNP_alt_ID_database_property_schema.py

```diff
@@ -2,56 +2,94 @@
 Author:  Samantha Piekos
 Date:  09/30/20
 Name:  format_dbSNP_alt_ID_database_property_schema.py
-Description: Write all unique databases that have alternative IDs for genetic variant for a .vcf or .txt input file
-as properties of GeneticVariant.
+Description: Write all unique databases that have alternative IDs for genetic 
+variant for a .vcf or .txt input file as properties of GeneticVariant.
 
 @file_input		input .vcf or .txt file of genetic variants
-@file_output	mcf output file recording all databases with alternative IDs for genetic variants as properties of GeneticVariant
+@file_output	mcf output file recording all databases with alternative IDs for
+				genetic variants as properties of GeneticVariant
 '''
 
 import sys
 import re
 
+def make_camel_case(item):
+	"""
+	Converts string with words seperated by '_' to camel case.
+
+	@item	string whose words are separated by '_'
+	@return	string that is in camel case
+	"""
+	camel_item = ""
+	l = item.split('_')
+	if len(l) > 2:
+		for i in l:
+			if len(i) > 2:
```

Comment:
should it be > 1? Or do you specifically only want to capitalize for 3 characters or more?

Reply:
There is weird formatting here in which there's _ used when it shouldn't be resulting in an error thrown. if it's greater than 2 this issue is avoided.